.github/workflows/docker-build-push-dockerhub.yml

```diff
@@ -16,7 +16,13 @@ on:
     types:
       - completed
   # Allow running workflow manually from the Actions tab
+  # workflow_dispatch:
   workflow_dispatch:
+    # Uncomment below to allow specific version workflow run
+    # inputs:
+    #   version:
+    #     description: 'Version to build'     
+    #     required: true
```

Comment:
It might be beneficial in the future to test the workflow run against specific versions for a more targeted debugging

Reply:
good with adding this commented. would be good to add an additional input to toggle pushing as well here